java/ql/src/Security/CWE/CWE-927/ImplicitPendingIntents.qhelp

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+
+<qhelp>
+
+<overview>
+<p>A <code>PendingIntent</code> describes an action in the form of an Intent that is intended to be given and executed 
+  at a later time by another application. The Intent wrapped by a <code>PendingIntent</code> is executed on behalf of 
+  the application that created it, and with its same privileges.</p>
```

Comment:
If I've understood this correctly, this could be more simply written as:

> A <code>PendingIntent</code> is used to wrap an <code>Intent</code> that will be supplied and executed by another application. When the <code>Intent</code> is executed, it behaves as if it were run directly by the supplying application, using the privileges of that application.